flask_app.py

```diff
@@ -118,7 +118,7 @@ def get_recordby_id():
     return data
 
 
-@app.route('/get_record_by_column_name', methods=['GET','POST'])
+@app.route('/get_record_by_column_name', methods=['GET'])
 @check_authorization
 def get_recordby_column_name():
     table_name = request.args.get('table_name')
```

Comment:
Do we have null or type checking?

Reply:
Added in the exception handling code reflected on this changeset